src/bin/pgcopydb/ld_replay.c

```diff
@@ -27,8 +27,10 @@
 typedef struct ReplayStreamCtx
 {
 	StreamApplyContext applyContext;
+	FILE *txnReplayBuffer;
 } ReplayStreamCtx;
```

Comment:
I think we should expose the current XID file name in that context structure, so that we can use the actual file name latter when applying from file. Using the symlink at apply-from-buffer time seems too “risky” to me, for no benefits.